src/components/Header/header.scss

```diff
@@ -0,0 +1,198 @@
+@import "../../styles/main.scss";
+
+.header {
+   background: rgba(241, 235, 235, 0.5);
+   height: 60px;
+   width: 100%;
+   text-align: center;
+   z-index: 500;
+   position: sticky;
+   top: 0;
+   display: flex;
+   align-items: center;
+   justify-content: center;
+   padding: 2px 5px;
+   box-sizing: border-box;
+   @media (max-width: 426px) {
+    height: 40px;
+  }
+      &__menu {
+        width: 54px;
+        height: 47px;
+        margin: 12px, 0px, 0px, 8px;
+        cursor: pointer;
+        &:hover {
+        width: 60px;
```

Comment:
Revisit tabulation in each scss files.
Also, as recommendation, install EsLint plugin for VSCode

Reply:
I have some problems with settings autoformating. Probably you can help me with it on the meeting, when we have time. Plugin is installed.